### ISSUE TITLE
New WCAG Criteria - Focus Not Obscured (Enhanced) - AAA

### ISSUE THREAD

**Issue by lpgoldstein** [DOI-ONRR/nrrd#3047]
Some of our focus indicators need to be stronger - might consider a two-color focus indicator. We are missing some focus indicators but we already have issues for that

**Comment by Maroyafaied** [comment]
This is for buttons and links, anything that needs to be clicked on .

**Comment by lpgoldstein** [comment]
No longer relevant
